src/quo2/components/info/lowest_price.cljs

```diff
@@ -0,0 +1,68 @@
+(ns quo2.components.info.lowest-price
+  (:require [quo.react-native :as rn]))
+
+(def centrify {:style {:flex-direction  :row
+                       :justify-content :center
+                       :align-items     :center}})
+
+(defn border-alignment [value-bg-color]
+  {:style {:align-self         :flex-start
+           :padding-horizontal 16
+           :padding-vertical   2
+           :flex-grow          0.25
+           :border-radius      4
+           :background-color   value-bg-color}})
+
+(def line-dots-props
+  {:ellipsize-mode  :middle
+   :number-of-lines 1
+   :font-size       10
+   :line-height     2})
+
+(defn dots
+  "Returns the dots"
+  [lots-o-dots?]
+  (let [dots-text " . . . . . . . . . . . . . . . . "]
```

Comment:
Sounds like many dots :D 

Probably it should be way more cleaner using something like `(str " " (clojure.string/join (repeat 16 ". ")))`

Also this can simplify the whole function

`(str " " (clojure.string/join (repeat (if lots-o-dots? 48 16) ". ")))`

Reply:
You can refer the join function on the requires as `[clojure.string :refer [join]]` to make it more readable

`(str " " (join (repeat (if lots-o-dots? 48 16) ". ")))`